HW6/Second_GitHub_Task.py

```diff
@@ -0,0 +1,7 @@
+user_login = input("Enter your login please:")
```

Comment:
Here's an optimized version of your code:

while True:
    user_login = input("Enter your login please: ")
    if user_login == "First":
        print("Thank you! We are glad to have you at our website, enjoy!")
        break
    else:
        print("Error! You wrote the wrong login. Please, try one more time.")